Portfolio/src/App.vue

```diff
@@ -138,43 +153,45 @@ body {
         list-style: none;
     }
 
-    .menu-fullName {
-        display: flex;
-        flex-direction: column;
-        align-items: flex-end;
-        font-size: 22px;
-        color: #ffffffa4;
-        margin: 3rem 2rem;
+    .menu-links a {
+        text-decoration: none;
+        color: #ffff;
+    }
```

Comment:
These kind of selectors makes your css brittle and makes it harder to maintain over time. (Speaking from experience, other people might have different opinions.)

Instead of using `a` to select all links within menu-links, then apply a className to each link instead. That way you decouple your css from your html structure and elements, meaning that you can easily replace out anchor tags with button tags or divs or spans or web components in the future, without having to change your styles.

Reply:
Done!